templates/store/search.html

```diff
@@ -52,74 +52,73 @@
   </div>
   <div class="row filter-row">
      <div class="col-4">
-       <ul class="p-inline-list entitity-search-selector">
-         <li class="p-inline-list__item {% if context.current_type == None %}is-selected{% endif %}"
-             data-type="all" tabindex="0" role="button">
-             All
-         </li>
-         <li class="p-inline-list__item {% if context.current_type == 'charm' %}is-selected{% endif %}"
-             data-type="charm" tabindex="0" role="button">
-             Charms
-         </li>
-         <li class="p-inline-list__item {% if context.current_type == 'bundle' %}is-selected{% endif %}"
-             data-type="bundle" tabindex="0" role="button">
-             Bundles
-         </li>
-       </ul>
+      <nav class="p-tabs">
+        <ul class="p-tabs__list entity-search-selector" role="tablist">
+          <li class="p-tabs__item" role="button">
+            <a href="#_" class="p-tabs__link" tabindex="0" role="button" aria-selected="{% if context.current_type == None %}true{% endif %}" data-type="all">All</a>
```

Comment:
You should add the if statement around the aria-selected otherwise unselected tabs will get `aria-selected=""`